Formula/xidel.rb

```diff
@@ -0,0 +1,40 @@
+class Xidel < Formula
+  desc "XPath/XQuery 3.0, JSONiq interpreter to extract data from HTML/XML/JSON"
+  homepage "https://www.videlibri.de/xidel.html"
+  url "https://github.com/benibela/xidel/releases/download/Xidel_0.9.8/xidel-0.9.8.src.tar.gz"
+  sha256 "72b5b1a2fc44a0a61831e268c45bc6a6c28e3533b5445151bfbdeaf1562af39c"
+  license "GPL-3.0-or-later"
+
+  livecheck do
+    url :stable
+    regex(%r{href=.*?/tag/Xidel_(\d+(?:\.\d+)+)["' >]}i)
+  end
+
+  head do
+    url "https://github.com/benibela/xidel.git", branch: "master"
+    resource "flre" { url "https://github.com/benibela/flre.git", branch: "master" }
```

Comment:
```suggestion
    resource("flre") { url "https://github.com/benibela/flre.git", branch: "master" }
```
You need parentheses if you want to wrap the block in `{`, `}`.